spec/index.html

```diff
@@ -191,22 +197,230 @@ <h2>Introduction</h2>
     <a href="#sec-grammar" class="sectionRef">Turtle Grammar</a>
     and <a href="#sec-parsing" class="sectionRef">Parsing</a>.
   </p>
+
+<section id="typographical-conventions" class="informative">
+  <h3>Typographical conventions</h3>
+
+  <p>The following typographic conventions are used in this specification:</p>
+
+  <dl class="typography">
+    <dt><code>markup</code></dt><dd>
+      Markup (elements, attributes, properties),
+      machine processable values (string, characters, media types),
+      property name,
+      or a file name is in red-orange monospace font.</dd>
+    <dt><dfn data-no-xref="">definition</dfn></dt><dd>
+      A definition of a term, to be used elsewhere in this or other specifications,
+      is in bold and italics.</dd>
+    <dt><a data-lt="definition">definition reference</a></dt><dd>
+      A reference to a definition <em>in this document</em>
+      is underlined and is also an active link to the definition itself. </dd>
+    <dt><a href="">hyperlink</a></dt><dd>
+      A hyperlink is underlined and in blue.</dd>
+    <dt>[<a href="">reference</a>]</dt><dd>
+      A document reference (normative or informative) is enclosed in square brackets
+      and links to the references section.</dd>
+    <dt><a href="" class="cp-terminal"><code>:</code></a></dt><dd>
+      A grammar literal terminal with specific Unicode code points.
+      Grammar literal terminals used in this specification include the following:
+
+      <!-- Selected unicode character descriptions, taken from https://en.wikipedia.org/wiki/List_of_Unicode_characters -->
+      <table class="cp-definitions">
```

Comment:
This table is out of proportion for the introduction and just before the non-normative descriptive section. It just slows the first time reader down.

Here, it mentions `e` for example, leaving the query why not `a`, `b`, ...

(table = "A grammar literal terminal ..." to end `|}`)

At a minimum - a multiple column table.
Better, is to put the table at the end of the grammar 6.6.

If a mention is needed (and I don't think it is) with something like "A list of terminal mentioned in the grammar is <here>"

Down text still has: `U+`

>The characters [-](https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/pull/46.html#cp-hyphen), [‿](https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/pull/46.html#cp-undertie), [⁀](https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/pull/46.html#cp-character-tie), and the combining diacritical marks (U+0300 to U+036F)

Reply:
> This table is out of proportion for the introduction and just before the non-normative descriptive section. It just slows the first time reader down.

Yes, I thought about this, and your suggestion (below) to add it to the grammar section makes sense to me.

> Here, it mentions `e` for example, leaving the query why not `a`, `b`, ...

That's because `e` (and `E` are explicitly referenced (similar to `0` and `9`). All Unicode characters could be listed, but only a relatively small subset are actually referenced.

> (table = "`A grammar literal terminal ..." to end `|}`)

I don't understand this comment.

> At a minimum - a multiple column table. Better, is to put the table at the end of the grammar 6.6.

👍 

> If a mention is needed (and I don't think it is) with something like "A list of terminal mentioned in the grammar is "
> 
> Down text still has: `U+`
> 
> > The characters [-](https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/pull/46.html#cp-hyphen), [‿](https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/pull/46.html#cp-undertie), [⁀](https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/pull/46.html#cp-character-tie), and the combining diacritical marks (U+0300 to U+036F)

The diacritical marks don't have an associated Glyph, so were left as bare code points.

Reply:
I wasn't suggesting removing the rest of the typography section. (I'm neutral as to whether to have it or not.)

Reply:
Although it may be generally useful, the motivation for putting it in was to have a place for the table, which isn't needed anymore. We can add something like the original back, although should probably do it universally. I personally don't think it's necessary.